### DISCUSSION TITLE
Article Widget

### DISCUSSION THREAD

**Discussion by mitchellcairns** [outline/outline#6529] [Ideas]
Ideally, it would be nice to have an article embed widget option.

Context, we sell products on Shopify, and being able to have installation articles/product guides embedded on the product page itself would be great (That are auto-updating).

**Comment by tommoor** [comment]
@mitchellcairns if you share a document publicly then you can embed that page in an iframe

**Reply by mitchellcairns** [reply]
Yes, I know, however it also reveals all the navigation etc.

**Reply by tommoor** [reply]
I'm not sure which navigation would be visible in this case, sharing a single document would not show the sidebar. You could also use the API to achieve this.

**Reply by mitchellcairns** [reply]
Thanks for that. The API seems like it would work. 

I have a question on that. The API docs say:
`Some API endpoints allow unauthenticated requests for public resources and they can be called without an API key.`

Which endpoints? Am I able to retrieve documents in this way? Otherwise I think I'd need to spin up a different web server to retrieve and keep the API key hidden.

**Reply by tommoor** [reply]
If shared publicly you can use the `shareId` instead of an api key to retrieve the content, the share id is the part after the `/s/` in the url. e.g:

<img width="1384" alt="image" src="https://github.com/outline/outline/assets/380914/d0639981-11ae-479f-931c-d96303ef435e">

**Reply by mitchellcairns** [reply]
That works beautifully. Thank you so much for your quick help!
